xbmc/cores/RetroPlayer/streams/RetroPlayerVideo.h

```diff
@@ -11,6 +11,8 @@
 #include "IRetroPlayerStream.h"
 #include "cores/RetroPlayer/RetroPlayerTypes.h"
 
+#include <cstdint>
```

Comment:
This file (and maybe others) also need `#include <stddef>` for the use of `size_t`. Can you also check for this?